core/scss/components/main-nav/_main-nav.scss

```diff
@@ -0,0 +1,162 @@
+@charset "UTF-8";
+
+//
+// Main Navigation
+//
+// The primary site navigation that is usually in the header region of a website. It is horizontal on larger devices and collapses on small devices.
+//
+// .su-main-nav--center - Centered main nav
+// .su-main-nav--right - Right-aligned main nav
+//
+// Markup: ../templates/components/main-nav/main-nav.twig
+//
+// Style guide: Components.Main Navigation
+//
+
+.su-main-nav {
+  @include font-smoothing(antialiased);
+  font-family: $font-sans;
+
+  ul {
+    @include margin(0 null);
+    list-style: none;
+  }
+
+  a {
+    text-decoration: none;
+    color: $color-black;
+  }
+}
+
+.su-main-nav__menu-level1 {
+  display: flex;
+  flex-direction: column;
+  background-color: $color-black;
+  width: 100%;
+
+  > li {
+
+    @include margin(null null 0);
+
+    a {
+      @extend %fancyhover;
```

Comment:
Oh, perhaps there is a change to refactor this so that we don't have to extend a css class. Can you make this a mixin with perhaps a more descriptive name.